tests/wpt/mozilla/tests/mozilla/mozbrowser/private_browsing.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf8" />
+<title>Private browsing</title>
+</head>
+<body>
+<iframe id="private_iframe"></iframe>
+</body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  async_test(function(t) {
+    var privateFrame = document.getElementById("private_iframe");
```

Comment:
We should create the iframe instead of using an existing one. Otherwise we could run into problems with the initial load of `about:blank`.